config-overrides.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  webpack: (config, env) => {
+    if (env === 'development') {
+      config.module.rules.push({
+        test: /\.(js|mjs|jsx|ts|tsx)$/,
```

Comment:
creo que no es tan necesario toda la expresión, no está mal pero ¿en algún momento estamos usando typescript?

Reply:
you are right, sera mejor si acoto laas extensiones